frontend/src/actionCreators/parcelsActionCreator.ts

```diff
@@ -126,7 +126,7 @@ export const createParcel = (parcel: IParcel) => async (dispatch: Function) => {
       parcel,
     );
     dispatch(success(actionTypes.ADD_PARCEL, status));
-    dispatch(storeParcelDetail(data));
+    dispatch(storeParcelDetail(null));
```

Comment:
It makes more sense to empty out the selected (active) property after it is added/updated since we close the sidebar whenever we do a db update.